docs/installation.rst

```diff
@@ -51,18 +51,22 @@ URL paths
 ~~~~~~~~~
 
 Include the required CAS URL endpoints in your project's root ``URLconf``
-with the following::
+in ``urls.py`` with the following::
 
+   from django.urls import include, re_path
+   
```

Comment:
This is usually already included in your project's url root urlconf, but I have no objection to include it.